src/main/java/com/aws/greengrass/shadowmanager/sync/RequestMerger.java

```diff
@@ -27,6 +32,9 @@
 public class RequestMerger {
     private static final Logger logger = LogManager.getLogger(RequestMerger.class);
 
+    @Setter
+    private Direction syncDirection = Direction.BETWEEN_DEVICE_AND_CLOUD;
```

Comment:
Does it mean it's the default of the sync direction? Is there any default initializer for the Direction class we can use instead of setting the value here?

Reply:
I had to add this here for unit tests. For some reason it was set to null for some reason if it was not set. It is not the case in integ tests since we will always set it when we do the subscriptions. We also have a default value in Shadow Manager recipe which should set it.